src/main/java/com/checkmarx/flow/custom/GitHubIssueTracker.java

```diff
@@ -343,15 +351,6 @@ private JSONObject getJSONCloseIssue() {
         return requestBody;
     }
 
-    /**
-     * @return Header consisting of API token used for authentication
```

Comment:
this function should control what to do, not be deleted.

Reply:
you speak about the "createAuthHeaders" method? 

As it is duplicated in multiple class, it is better to have a real centralized auth manager.